admin/index.php

```diff
@@ -34,7 +24,7 @@
 <?php
```

Comment:
If you remove the 
```php
    <?php
    require_once(__DIR__ . "/../css_binding.php");
    ?>
```
It fixes the admin page altogether.
It seems the two imports are clashing with each other.